.github/workflows/test.yaml

```diff
@@ -75,7 +81,9 @@ jobs:
 
         - name: Run Fork Tests
           # Only run fork tests if not from a forked repo (Else it always fails)
-          if: github.event.pull_request.head.repo.full_name == github.repository
+          if: |
+            github.event_name == "pull_request" ||
```

Comment:
I am hoping this has short-circuit eval because `pull_request` token fails when merging to main